ethereum_clients/Plugin.js

```diff
@@ -180,7 +180,14 @@ class Plugin extends EventEmitter {
 
   async start(release, flags, config) {
     // TODO do flag validation here based on proxy metadata
-
+    if (this.config.beforeStart) {
+      if (this.config.beforeStart.execute) {
```

Comment:
nit: shed one layer by combining: `if (this.config.beforeStart && this.config.beforeStart.execute)`

Reply:
thx